assets/css/_route_ladder.scss

```diff
@@ -49,21 +49,35 @@
 
   .card-body {
     display: inline-grid;
-    grid-template-columns: 1fr 1fr auto 1fr 1fr;
+    grid-template-columns: 1fr auto 1fr;
     padding: 6px 8px;
-    align-items: center;
+  }
+
+  .c-route-ladder__header__action-container {
+    display: inline-flex;
+    div {
+      padding-right: 0.5rem;
+    }
   }
 
   .c-route-ladder__dropdown-button {
     height: 2.25rem;
     width: 2.25rem;
+    // TODO All of this is bad and should be reverted when we fix the left nav button colors
+    &.btn-primary {
+      --bs-btn-bg: #451b83;
```

Comment:
Nit (non-blocking): I think these colors can be imported from the `tokens_2021` file. 

At the top of the file, you could do something like:
```css
@use "./color/tokens_2021" as colors;
```

And then `#451b83` becomes `colors.$eggplant-700` and `#260f54` becomes `colors.$eggplant-800`.

This is _very_ temporary, so not a huge deal either way.

Reply:
Yeah, since these will be deleted, I specifically didn't add an extra line of code for the import, so that the single chunk can be deleted.

Reply:
Ah yeah that makes sense!